### DISCUSSION TITLE
Kubernetes & (Sealed) Secrets

### DISCUSSION THREAD

**Discussion by zubcevic** [WebGoat/WebGoat#1386] [Ideas]
Perhaps we (or I) can make a lesson on secrets in Kubernetes and how Sealed Secrets or using a CyberArk/Conjur approach can make the use of secrets more secret

**Comment by aolle** [comment]
Do you have in mind an theoretical lesson, a hands on lesson or both? I think that any topic around prod sec is welcome, isn't it?
Let me know if I can help you in any thing or if we can work together with that. I also have some experience with Conjur.
Regards and thank you.

**Reply by zubcevic** [reply]
I think at least some theoretical part which raises awareness of what to do with secrets in a GitOps/ArgoCD kind of approach and some practical things/exercises. This week I am discussing some key exchange process where KubeSeal is used externally. I can set up a branch for this and make some outline. Which we could then work on together.
